src/parse.js

```diff
@@ -341,50 +341,24 @@ export class TemplateCellParser extends CellParser {
 }
 
 // This is our custom override for parsing a template that allows
-// backticks. Based on acorn's readTmplToken.
+// backticks. Based on acorn's readInvalidTemplateToken.
 function readTemplateToken() {
-  let out = "", chunkStart = this.pos;
-  for (;;) {
-    if (this.pos >= this.input.length) {
-      out += this.input.slice(chunkStart, this.pos);
-      return this.finishToken(tt.template, out);
-    }
-    let ch = this.input.charCodeAt(this.pos);
-    if (ch === 36 && this.input.charCodeAt(this.pos + 1) === 123) { // '${'
-      if (this.pos === this.start && this.type === tt.template) {
-        this.pos += 2;
-        return this.finishToken(tt.dollarBraceL);
-      }
-      out += this.input.slice(chunkStart, this.pos);
-      return this.finishToken(tt.template, out);
-    }
-    if (ch === 92) { // '\'
-      out += this.input.slice(chunkStart, this.pos);
-      if (this.pos < this.input.length - 1) out += this.readEscapedChar(true);
-      else ++this.pos;
-      chunkStart = this.pos;
-    } else if (isNewLine(ch)) {
-      out += this.input.slice(chunkStart, this.pos);
-      ++this.pos;
-      switch (ch) {
-      case 13:
-        if (this.input.charCodeAt(this.pos) === 10) ++this.pos; // falls through
-      case 10:
-        out += "\n";
-        break;
-      default:
-        out += String.fromCharCode(ch);
+  out: for (; this.pos < this.input.length; this.pos++) {
+    switch (this.input[this.pos]) {
+      case "\\": ++this.pos; break;
```

Comment:
maybe we need to call `finishToken` in this case if there isn't another character coming next and not increment?

Reply:
was comparing this to https://github.com/acornjs/acorn/blob/556a7ab62dd999c9024ed0e0920ebb5f6521c4de/acorn/src/tokenize.js#L672-L692

but that doesn't seem to be necessary since the boundaries of the `for` loop already enforce that

Reply:
I will tack on a fix for the out of bounds with trailing slash. Thanks for tracking it down!